README.md

```diff
@@ -1,8 +1,50 @@
-# Final-Project-Template
+# The Rise of EVs and their Impact on the Environment
 <!-- Edit the title above with your project title -->
 
 ## Project Overview
 
+### Topic:
+The rapid increase in electric vehicles (EV) available on the road has offered a very promising solution to global warming efforts around the world. However, understanding the impact of EVs on the environment starts with the full lifecycle starting from the manufacturing to usage on the road and inevitably disposal. This topic is important because it seems that people ar shifting over to EVs for everyday commuting due to the financial benefits and reliefs that may come in the form of gas and maintenance. This paired with the rising concern in the environment and global warming brings it fairly high on the relevance charts.
```

Comment:
The topic seems slightly vague. Is the goal to understand the environmental impact of EV use and production? or to examine the full range of the benefits EV's provide (environmental, financial and ease of use)